tests/dummy/app/templates/snippets/buttons-dropdown.hbs

```diff
@@ -0,0 +1,9 @@
+{{md-btn-dropdown text='Button' activates="dropdown-target"}}
```

Comment:
I think that the most reasonable way to build this component expressively is something like

```hbs
{{#md-btn-dropdown text='My Button'}}
  <li>{{link-to 'Go Home' 'index'}}</li>
  ...
{{/md-btn-dropdown}}
```

This way the component handles wiring up the dropdown its self. I'm not crazy about exposing the concept of "activates=" to consumers, since ultimately the hope is to get rid of the materialize javascript and replace it with a pure ember solution.